META.json

```diff
@@ -42,8 +43,9 @@
       },
       "runtime" : {
          "requires" : {
-            "Minion" : "1.15",
-            "MongoDB" : "0"
+            "Minion" : "9.00",
+            "MongoDB" : "2.0",
+            "Mojolicious" : "8.10"
```

Comment:
I have not found release Mojolicious with version 8.10, so all tests now fail (https://travis-ci.org/avkhozov/Minion-Backend-MongoDB/jobs/469083047)

Reply:
The problem was that when minion worker forks, mongodb client needs to reconnect

https://metacpan.org/pod/MongoDB::MongoClient#THREAD-SAFETY-AND-FORK-SAFETY

So I need to intercept the worker forks. 

https://github.com/mojolicious/minion/issues/75

To do this, Mojo maintainer adds an new emit to IOLoop that is available in the pre-release. 

https://github.com/mojolicious/mojo/commit/2edfdb2d2d53f5cd3a0c4609bbe1f9dcccd719ac

I don't know when 8.10 will be released.

Reply:
They are waiting for this https://github.com/mojolicious/mojo/pull/1301